documentation/pixelclassification/pixelclassification.md

```diff
@@ -97,6 +102,21 @@ To access the "Import Labels" feature in the GUI, do the following:
 
 <a href="snapshots/labels_context_menu.png" data-toggle="lightbox"><img src="snapshots/labels_context_menu.png" class="img-responsive" /></a>
 
+## Automatic feature suggestion {#suggest}
+If you are not confident in your choice of pixel features, **Suggest Features** functionality might help. Press the button and a new dialog will pop up.
+
+<a href="snapshots/suggest_features_button.png" data-toggle="lightbox"><img src="snapshots/suggest_features_button.png" class="img-responsive" /></a>
+
+<a href="snapshots/suggest_features_dialog.png" data-toggle="lightbox"><img src="snapshots/suggest_features_dialog.png" class="img-responsive" /></a>
+
+
+It works by evaluating the classifier predictions using different feature sets on the pixels you have already annotated. The so called "out-of-bag" predictions are used, so for every tree in the Random Forest we only use the pixels it has not seen in training to estimate the error. Still, even though the pixels were not seen in training, randomization is done on the pixel level, so the tree has likely seen some similar pixels and the error estimate is overly optimistic. Besides the error, ilastik also reports the computing time which can be helpful when trying to find the balance between runtime and accuracy. By choosing the feature set in the lower left corner, you can also see the predictions ilastik would make with these features on the current field of view.
+
+<a href="snapshots/suggest_features_results_2.png" data-toggle="lightbox"><img src="snapshots/suggest_features_results_2.png" class="img-responsive" /></a>
+
+If you like one of the suggested features sets better than your current selection, you can replace your selection with the a new set by pressing the **Select Feature Set** button. 
```

Comment:
there's a loose extra "the" in "(...) with the a new set"

Reply:
```suggestion
If you like one of the suggested features sets better than your current selection, you can replace your selection with the new set by pressing the **Select Feature Set** button. 
```